src/CoreWCF.Http/tests/CoreWCF.Http.Tests.csproj

```diff
@@ -6,18 +6,24 @@
     <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
     <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+
+    <!-- Required for multi-framework support for in-memory integration testing-->
```

Comment:
Can you explain this? I can run the tests targeting .NET Framework with it being a .dll, you just need to use the dotnet.exe tool to launch it. Or is this for some other purpose?

Reply:
Yes, the `TestServer` is implemented slightly differently between framework versions and versions of Visual Studio. If you remove those lines in Visual Studio 2019 latest, it won't compile without `GenerateProgramFile=false`, which is in this case a redundant normalizing for test SDK versions. Ditto `OutputType`. `PreserveCompilationContext` is necessary for `TestServer` to work in all flavours.